### ISSUE TITLE
Broken bpftrace with clang 10.0

### ISSUE THREAD

**Issue by SchrodingerZhu** [archlinuxcn/repo#1582]
### 问题类型 / Type of issues

* 打包错误 / packaging error

### 受影响的软件包 / Affected packages

* bpftrace 0.10.0-2 
----
### Reason

Broken with an update of `clang` to version `10.0`. The error is shown as the following:
```
/usr/bin/bpftrace: error while loading shared libraries: libclang.so.9: cannot open shared object file: No such file or directory
```

**Comment by attenuation** [comment]
I have triggered a rebuild task, wait for the next edition compiled automatically

**Comment by SchrodingerZhu** [comment]
fixed with `bpftrace-0.10.0-4`
